### PR TITLE
Transition to structural schema required in 1.16

### PR DESCRIPTION
**Pull request by alewitt2**
this PR allows for old non structural schema under deploy.razee.io/v1alpha1 and continues support the schema in the code as well as define the new structural schema under deploy.razee.io/v1alpha2 and also understand the new schema in the code.

### CODE REVIEW

File: src/FeatureFlagSetLDController.js [hunk 1]
```diff
@@ -30,34 +30,46 @@ module.exports = class FeatureFlagSetLDController extends BaseController {
...
     }
     if (!this._sdkkey) {
-      throw Error('spec.sdk-key must be defined');
+      throw Error('A LaunchDarkly SDK Key must be defined');
     }
     let sdkkey = this._sdkkey;
+    this._sdkkeyHash = hash(sdkkey);
 
     let client;
     if (clients[sdkkey]) {
```
Comment (adamkingit) [inline src/FeatureFlagSetLDController.js]: Simplify this if/else

File: .travis.yml [hunk 2]
```diff
@@ -35,15 +35,6 @@ deploy:
     on:
       tags: true
       condition: ${TRAVIS_TAG} =~ ^[0-9]+\.[0-9]+\.[0-9]+_[0-9]{3}$
-  - provider: releases
```
Comment (adamkingit) [inline .travis.yml]: Why stop publishing the yaml for draft releases?

Comment (alewitt2) [reply .travis.yml]: didnt find it useful

File: src/FeatureFlagSetLDController.js (hunk 1 above)
Comment (adamkingit) [reply src/FeatureFlagSetLDController.js]:
```javascript
    let client=objectPath.get(clients, [sdkkey, 'client']);
    if (client===undefined) {
      client = LaunchDarkly.init(sdkkey);
      await client.waitForInitialization();
      objectPath.set(clients, [sdkkey, 'client'], client);
    }
    objectPath.set(clients, [sdkkey, 'instances', this._instanceUid], true); ```

File: package.json [hunk 3]
```diff
@@ -17,6 +17,7 @@
   ],
   "scripts": {
     "start": "node src/index.js",
+    "start-cws": "node src/conversionWebhookServer.js",
```
Comment (adamkingit) [inline package.json]: remove

**Pull request merged**